doc/users/next_whats_new/updated_scatter.rst

```diff
@@ -0,0 +1,15 @@
+Enable configuration of empty markers in `~matplotlib.axes.Axes.scatter`
+------------------------------------------------------------------------
+
+`~matplotlib.axes.Axes.scatter` can now be configured to plot empty markers without additional code. Setting ``facecolors`` to *'none'* and defining ``c`` now draws only the edge colors for fillable markers.
```

Comment:
I think this should more explicitly highlight the "with colormap" part of this change, it has always been possible to have empty single color markers with `facecolor='none', edgecolor='C0'`, e.g.

Admittedly it is a bit surprising that specifying `edgecolor` explicitly was required, but that is due to the default being to follow "face"

Does this PR do anything to make that behavior change (without a cmap?)

Reply:
We've changed the text in 39cfd56, hopefully now it's more fitting. If I understand your question correctly, yes, this PR does change that behavior: for the value of `edgecolor`, `c` now takes precedence over `face`